kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/challenges/OrderDTO.java

```diff
@@ -0,0 +1,23 @@
+package com.kodilla.good.patterns.challenges;
+
+public class OrderDTO
```

Comment:
jeśli to ma być immutable to klasa też powinna być final

Reply:
Z tego co pamiętam z naszej rozmowy, to DTO nie powinno być final więc zostawiam jak jest